src/drivers/libinput/lv_libinput_private.h

```diff
@@ -18,6 +18,8 @@ extern "C" {
 
 #if LV_USE_LIBINPUT
 
+#include "lv_xkb_private.h"
```

Comment:
is this include correct?
Including xkb headers independent of value LV_LIBINPUT_XKB?

Reply:
Yes it would be ok because `lv_xkb.h` will not try to include `xkbcommon` or declare anything unless `LV_LIBINPUT_XKB` is `1`.

But for clarity and consistency, I have pushed a change :+1: